### DISCUSSION TITLE
Pi-hole LXC with optional Unbound add-on

### DISCUSSION THREAD

**Discussion by fifolifo** [tteck/Proxmox#3998] [General] [answered]
I wanted to report what I believe to be an error in the /etc/unbound/unbound.conf.d/pi-hole.conf file. The interface in the conf file is 0.0.0.0 when it should be 127.0.0.1.  I came across this when I noticed the Pi-hole query log had entries of "(answered by dns.google#53)" and not "(answered by localhost#5335)" as I get from my older pi-hole that I installed a while ago on a raspberry pi.  

Thanks so much for your Proxmox scripts.  They are really great.

**Comment by tteck** [comment, marked as answer]
Thanks, https://github.com/tteck/Proxmox/commit/2d3cae43cbf679a8171c68ef2dd72e13d521cd74

**Comment by barisahmet** [comment]
I am still seeing "answered by dns.google#53" on a fresh install.
